src/lib.rs

```diff
@@ -58,33 +58,40 @@ pub fn cli() -> Command {
                 .required(false)
                 .num_args(0..)
                 .value_delimiter(',')
-                .value_parser(clap::value_parser!(String))
+                .value_parser(clap::value_parser!(String)),
         )
         .arg(
             Arg::new("since")
                 .help("a timestamp, events must be newer than this to pass")
                 .long("since")
                 .required(false)
                 .num_args(1)
-                .value_parser(clap::value_parser!(u32))
+                .value_parser(clap::value_parser!(u32)),
         )
         .arg(
             Arg::new("until")
                 .help("a timestamp, events must be older than this to pass")
                 .long("until")
                 .required(false)
                 .num_args(1)
-                .value_parser(clap::value_parser!(u32))
+                .value_parser(clap::value_parser!(u32)),
         )
         .arg(
             Arg::new("limit")
                 .help("maximum number of events to be returned in the initial query")
                 .long("limit")
                 .required(false)
                 .num_args(1)
-                .value_parser(clap::value_parser!(u32))
+                .value_parser(clap::value_parser!(u32)),
+        )
+        .arg(
+            Arg::new("count")
+                .help("maximum number of events to be returned in the initial query")
```

Comment:
the help text needs to be updated from `maximum number of events to be returned in the initial query` to something suitable for the `COUNT` flag.

Reply:
I'm not native English speaker. So this change might be not good. Please let me known if it so.